src/client/config/openfin/demo.app.json

```diff
@@ -3,7 +3,7 @@
     "splashScreenImage": "https://web-demo.adaptivecluster.com/images/splash-screen.jpg",
     "startup_app": {
         "name": "Reactive Trader Cloud",
-        "url": "https://web-demo.adaptivecluster.com",
+        "url": "http://web-dev.adaptivecluster.com/",
```

Comment:
why these changes ?
1) https ==> http
2) demo ==> dev

Reply:
I don't know and the other guys are on holidays